pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -251,7 +251,9 @@ public static double getTotalNicLimit(List<String> nics, BitRateUnit bitRateUnit
             try {
                 return readDoubleFromFile(getReplacedNICPath(NIC_SPEED_TEMPLATE, nicPath));
             } catch (IOException e) {
-                log.error("[LinuxInfo] Failed to get total nic limit.", e);
```

Comment:
I think this [config](https://github.com/apache/pulsar/blob/9edaa8569deff9c0cbb41b261fee472603f3df4d/conf/broker.conf#L1349) should be helpful for you.

Reply:
The configuration item `loadBalancerOverrideBrokerNicSpeedGbps` is used to override the auto-detection of the network interface max speed.
Actually, the system (for example, in a scenario with multiple NICs) can read the max speed of these NICs.
On the other hand, the network interface max speed of each broker node may be different. If a fixed value is manually configured to overwrite the automatic detection mechanism, the problem handling becomes complex.
 I think using the configuration item is only a workaround, not the best way to solve the problem.